client/src/components/Explorer/ExplorerPanel.js

```diff
@@ -137,19 +137,21 @@ class ExplorerPanel extends React.Component {
 
     return (
       <FocusTrap
-        tag="nav"
+        tag="div"
+        role="dialog"
         className="explorer"
         paused={paused || !page || page.isFetching}
         focusTrapOptions={{
-          initialFocus: '.c-explorer__close',
+          initialFocus: '.c-explorer__header',
           onDeactivate: onClose,
         }}
       >
-        <Button className="c-explorer__close u-hidden" onClick={onClose}>
+        <Button className="c-explorer__close visuallyhidden" onClick={onClose}>
```

Comment:
The fact that this button is completely hidden is a bug – it was meant to be displayed on mobile only. I think this would be worth addressing separately so I’ve made a separate issue (#5396) about this. In the meantime, could you revert this back to being `u-hidden`?

I quite like the idea of having it visually hidden, but since this is a focusable element it will actually become visible when people tab through the explorer, which doesn’t look that great.

Reply:
Reverted in https://github.com/wagtail/wagtail/pull/5394/commits/03d9ad9989eca64b339ac8747ff7433e42286e0d